home.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>YoutTube - Custome</title>
+</head>
+<body>
+    <ul class="header">
+        <li ><a href="#">YoutTube</a></li>
+        <li style="float: right;"><a href="#">LogOut</a></li>
+    </ul>
+    <div class="side-navigation">
```

Comment:
- [ ] Instead of using `div` tag for side navigation it will be better if you use `asid` tag. 👍

Reply:
- [ ] Kindly note to always include an explanation, a "why", for each requested change. For example:

- This is a well-explained request:
it will be better if you use asid tag instead of using div tag for side navigation to improve our structure and semantics. 
- This request lacks a "why":
Instead of using div tag for side navigation it will be better if you use asid tag.

Reply:
- [ ] I think nstead of using `div` tag for side navigation it will be better if you use the `aside` tag. 👍  Because it will improve your code structure and semantics